soc/v2/aspeed/Kconfig.soc

```diff
@@ -3,15 +3,10 @@
 # Copyright (c) 2021 ASPEED Technology Inc.
 
 config SOC_FAMILY_ASPEED
-	select PLATFORM_SPECIFIC_INIT
 	bool
-
-if SOC_FAMILY_ASPEED
+	select PLATFORM_SPECIFIC_INIT
```

Comment:
line should go here: https://github.com/zephyrproject-rtos/zephyr/blob/2c274b411ea11b8ccdf1f17bb52693ac2682a2bd/soc/v2/aspeed/Kconfig

Reply:
fixed